### PR TITLE
Routing on one of multiple selected checkbox answers

### PR DESCRIPTION
**Pull request by jonnyshaw89**
### What is the context of this PR?
Added a contains condition to routing rules.
When a routing rule is specified on a checkbox, the routing rule checks if a value is one of the selected answers.

Fixes https://github.com/ONSdigital/eq-survey-runner/issues/684

### How to review 
On the census-household questionnaire test the routing of the employment-type question

### CODE REVIEW

**Comment by collisdigital** [comment]
Also needed on the which passport screen I think

**Comment by collisdigital** [comment]
This also fixes some issues reported by yogi, can we add them to the description?

**Comment by collisdigital** [comment]
https://github.com/ONSdigital/eq-survey-runner/issues/684

**Comment by jonnyshaw89** [comment]
@collisdigital i can't add this to the passports question as the question it routes to is added in here https://github.com/ONSdigital/eq-survey-runner/pull/683 so need this to go in then i can merge into the other branch

**Comment by yoganandkunche** [comment]
@jonnyshaw89 Re-tested the issue #684 one scenarios seems to be not working. Lets discuss on Monday

**Comment by jonnyshaw89** [comment]
@yoganandkunche i have corrected the routing on the none of the above option

**Comment by yoganandkunche** [comment]
@jonnyshaw89 As discussed selecting the 7 option along with any/all the 1-6 options must skip the next questions

**Comment by yoganandkunche** [comment]
Not merged into master yet. So will be testing it in pre-prod tomorrow as I will be busy with e2e testing today

File: app/questionnaire/rules.py [hunk 1]
```diff
@@ -8,19 +8,15 @@ def evaluate_rule(when, answer_value):
...
     condition = when['condition']
 
-    if isinstance(answer_value, list):
-        if len(answer_value) == 1:
-            answer_to_test = answer_value[0]
-        else:
-            return False
-    else:
-        answer_to_test = str(answer_value)
+    answer_to_test = str(answer_value) if 'equals' in condition else answer_value
```
Comment (iwootten) [inline app/questionnaire/rules.py]: 'equals' in condition 
or

    condition == 'equals'?

**Review by iwootten — CHANGES_REQUESTED** [review]

File: app/questionnaire/rules.py (hunk 1 above)
Comment (iwootten) [reply app/questionnaire/rules.py]: Ah is this converting everything with 'equals' in the title to a string? If so, then you probably should be explicit about it:

     if condition in ['equals', 'not equals']

**Review by iwootten — CHANGES_REQUESTED** [review]

File: app/questionnaire/rules.py (hunk 1 above)
Comment (samiwel) [reply app/questionnaire/rules.py]: I agree with Ian's comment here. It's wasn't immediately obvious that you're matching `equals` and `not equals` conditions. Probably better to make that explicit.

**Review by samiwel — COMMENTED** [review]
Looks good to me.
Do any of the census schemas have `contains` routing where an input field is displayed upon selecting Other?  I didn't see any examples of that in this PR. Are these all the cases?

**Review by samiwel — APPROVED** [review]

**Review by iwootten — APPROVED** [review]

**Pull request merged**